### ISSUE TITLE
Message-parameter for MethodStubVerificator

### ISSUE THREAD

**Issue by Hirschfuchs** [TypeStrong/ts-mockito#40]
As a developer I want to set a custom message in a "_verify_"-command. If the verification fails, I want this message to be displayed (like with _expect_ commands in Chai or with https://github.com/mattphillips/jest-expect-message/tree/main)

**Comment by roypeled** [comment]
Good idea, here it is: https://github.com/TypeStrong/ts-mockito/pull/46

**Comment by roypeled** [comment]
fixed in V3
